tests/js/spec/views/projectDetail/projectLatestReleases.spec.jsx

```diff
@@ -95,6 +97,7 @@ describe('ProjectDetail > ProjectLatestReleases', function () {
           projectSlug={project.slug}
           location={router.location}
           projectId={project.slug}
+          isProjectStabilized
```

Comment:
should we also test when `isProjectStabilized` equals false?

Reply:
yep, I meant to add them in a follow-up but pushed them here now 👍